Create PFS CodeSystem as an NHS England IG Asset

adrianhand2 opened the pull request:
IOPS-2261 https://nhsd-jira.digital.nhs.uk/browse/IOPS-2261 
code system required by GP Connect PFS Prescriptions API

SheenaSharma committed:
Create PFS CodeSystem as an NHS England IG Asset

adrianhand2 committed:
Merge d07729c0bd1167b89f105bd4f213028565b94771 into bffe6a8aebd73a41196055b64eaddf89888614bf

ryma2fhir committed:
GitHub Action: Find and Replace

digitalhealthmartin commented on codesystems/CodeSystem-England-PrescriptionOrderingParameters.xml:
As this is a CodeSystem, I wonder if we might want a more detailed name for the filename. For example, CodeSystem-England-PFSPrescriptionOrderParameter (have also made it singular instead of plural like previous assets too). If this changes occurs, then url, name and title would also need updating).

adrianhand2 replied:
I'm fine with that name change

digitalhealthmartin commented on codesystems/CodeSystem-England-PrescriptionOrderingParameters.xml:
```diff
@@ -0,0 +1,33 @@
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A code set to define parameters provided when managing patients' prescription" />
```
Suggest description to be 'A set of codes that define parameters provided when managing a patient's prescription'. (a change to the beginning an also updated patients' to patient's)

adrianhand2 replied:
happy with that if it's more consistent with other assets.

KazeemHamzat commented on codesystems/CodeSystem-England-PrescriptionOrderingParameters.xml:
```diff
@@ -0,0 +1,33 @@
+  <name value="EnglandPrescriptionOrderingParameters" />
+  <title value="England Prescription Ordering Parameters" />
+  <status value="draft" />
+  <date value="2023-11-01" />
```
It may be necessary to modify the date to today's date, to ensure that the created date falls within Sprint 2 period.

adrianhand2 replied:
will update

adrianhand2 committed:
IOPS2261 name, date and description updated

Rename of file, id, url, name and title.
date changed to 2023-11-20
description value changed to suggested text.

KazeemHamzat reviewed (approved):
Asset ID changed to CodeSystem-England-PrescriptionOrderingParameter.
